views/install.sh.erb

```diff
@@ -332,7 +334,7 @@ do_curl() {
 # do_fetch URL FILENAME
 do_fetch() {
   echo "trying fetch..."
-  fetch -o "$2" "$1" 2>$tmp_dir/stderr
+  fetch -T $download_timeout -o "$2" "$1" 2>$tmp_dir/stderr
```

Comment:
This is sort of lightly documented and not clear what the timeout encompasses and if it encompasses download time or is just DNS+Connect or what.